components/centraldashboard/public/components/main-page.js

```diff
@@ -106,6 +106,8 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
                 // eslint-disable-next-line max-len
                 computed: 'computeShouldFetchEnv(registrationFlow, workgroupStatusHasLoaded)',
             },
+            matchingIndex: Number,
```

Comment:
Why did you need to add this to the properties? IIUC it is only used from the `_setActiveMenuLink` method

Reply:
I answerer that question in a below window.